src/clj/puppetlabs/services/watcher/filesystem_watch_service.clj

```diff
@@ -0,0 +1,45 @@
+(ns puppetlabs.services.watcher.filesystem-watch-service
```

Comment:
Oh.  Interesting.  To start, I followed the pattern in file sync, and then remove an `enterprise` level when I moved the code into this repo.

Kinda weird, though, to have things in `puppetlabs.trapperkeeper` which aren't part of Trapperkeeper itself.  But I guess that ship has sailed.